tests/ApiPort.Tests/ApiPort.Tests.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp1.0</TargetFramework>
+    <RunCodeAnalysis>false</RunCodeAnalysis>
```

Comment:
Do we skip CodeAnalysis for all the other test projects too?

Reply:
No - just for netcoreapp1.0 types. I've moved the `RunCodeAnalysis` setting to a conditional on `TargetFramework==netcoreapp1.0` in the `Directory.Build.props`